ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -723,3 +742,21 @@ Download Prompts Weights In PVC
     Wait For Pods To Be Ready    label_selector=name=${model_name}-download-prompts    namespace=${namespace}
     Wait For Pods To Succeed    label_selector=name=${model_name}-download-prompts    namespace=${namespace}
     ...    timeout=${download_timeout}
+
+Get KServe Default Deployment Mode From DSC
+    [Arguments]    ${dsc_name}=${DSC_NAME}
+    ${rc}    ${mode}=    Run And Return Rc And Output
+    ...    oc get dsc ${DSC_NAME} -ojsonpath='{.spec.components.kserve.defaultDeploymentMode}'
+    Run Keyword And Continue On Failure    Should Be Equal As Integers    ${rc}    ${0}    msg=${mode}
+    IF    "${mode}" == "${EMPTY}"
```

Comment:
i think you need to add or condition also here to check if the deployment mode is set to Serverless or not

Reply:
why?

Reply:
explain in the chat :)